serial.py

```diff
@@ -22,12 +23,17 @@ class SerialDisconnectException(SerialException):
 
 class Serial:
 
-    BAUD_MAP = { 9600: termios.B9600, 115200: termios.B115200 }
+    BAUD_MAP = {
```

Comment:
Can you please split baud rate additions to a separate commit, as they're not related to the timeout changes?

Reply:
sure, will make another pull request